src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -139,29 +147,17 @@ def get_size(image_size, size, max_size=None):
             width = int(np.ceil(width / self.size_divisibility)) * self.size_divisibility
 
         size = (width, height)
-        rescaled_image = self.resize(image, size=size)
-
-        has_target = target is not None
+        image = self.resize(image, size=size)
```

Comment:
You probably need to add the `resample` argument here

Reply:
Still need to address this comment

Reply:
thanks, the default should be the correct one

Reply:
Why aren't you adding `resample=self.resample` here?

Reply:
correct, added it! (thanks 🙏 )